src/documentation/runners/apex.md

```diff
@@ -2,8 +2,113 @@
 layout: default
 title: "Apache Apex Runner"
 permalink: /documentation/runners/apex/
+redirect_from: /learn/runners/apex/
 ---
 # Using the Apache Apex Runner
 
-This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
+Apex‐Runner is a Runner for Apache Beam which executes Beam pipelines with Apache Apex as underlying engine. The runner has broad support for the Beam model and supports streaming and batch pipelines.
```

Comment:
The Apex Runner executes Apache Beam pipelines using `[Apache Apex](link)` as an underlying engine.